.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,32 @@
+name: NPM Publish
+
+on:
+  push:
+    tags:
+      - '*.*.*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
```

Comment:
Possibly bump to LTS

Reply:
I've switched to 14. Will see if there's any impact